### DISCUSSION TITLE
更新时遇到一个报错

### DISCUSSION THREAD

**Discussion by windy1021** [1715173329/UnblockNeteaseMusic#1137] [Q&A]
### Bug 描述

部署到本地进行更新时有一个报错，不知道是否完成更新
好像说yarn.lock这个文件被修改过，但我没有动过呀，不过版本号好像是对的？现在这种情况算更新完了吗？我还要进行什么操作吗，小白不太懂，麻烦大佬看看

### 日志内容

remote: Enumerating objects: 17, done.
remote: Counting objects: 100% (17/17), done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 17 (delta 10), reused 17 (delta 10), pack-reused 0
Unpacking objects: 100% (17/17), 15.30 KiB | 188.00 KiB/s, done.
From https://github.com/UnblockNeteaseMusic/server
   35f1fae..48beea2  enhanced   -> origin/enhanced
   b8d5f4b..8c06a88  master     -> origin/master
 * [new tag]         v0.27.2    -> v0.27.2
error: Your local changes to the following files would be overwritten by merge:
        yarn.lock
Please commit your changes or stash them before you merge.
Aborting
Updating 35f1fae..48beea2

### 网易云音乐版本号

2.10.8

### 操作系统

win11

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [X] 我确认我已经启用了 HTTPS 端口。
- [X] 我确认我已经正确设置了 EndPoint。
- [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by 1715173329** [comment]
重置当前更改再 pull。自行查阅 git 文档。
